frontend/src/app/models/groupMembers.ts

```diff
@@ -0,0 +1,7 @@
+import User from './user';
```

Comment:
You can just put this class inside `models/group.ts`. It's fine if we put similar models inside a single class as long as it doesn't get too messy.